### ISSUE TITLE
igraph_version() reports wrong version in 1.3.5

### ISSUE THREAD

**Issue by szhorvat** [igraph/rigraph#583]
**Describe the bug**

igraph_version() reports wrong version in 1.3.5

Also, you should push the tag @ntamas, and add pushing the tag to the release checklist :-)

**To reproduce**

```
> packageVersion('igraph')
[1] ‘1.3.5’
> igraph_version()
[1] "1.3.4.9022"
```

**Version information**

 - R/igraph version: 1.3.5, CRAN binary
 - R version: 4.2.1
 - Operating system: macOS

**Comment by ntamas** [comment]
Something must have gone wrong during the build process but I don't know what. Right now `IGRAPH_VERSION` in `igraph_version.h` is replaced on-the-fly with the R version number when we run `make`. I remember there was some manual hackery when I made the release last time and it was not entirely from a clean slate so something must have happened there. I'll check whether there's an easy workaround once I manage to figure out what went wrong.

**Comment by ntamas** [comment]
Hopefully c968d110 will make it harder to publish a tarball on CRAN that has an invalid hardcoded version number in `igraph_version()`. This will be released with 1.3.6.
